server/ngwmn/tests/test_views.py

```diff
@@ -24,23 +23,37 @@ class TestWellPageView(TestCase):
     def setUp(self):
         self.app_client = app.test_client()
         _agency_cd = 'DOOP'
-        _location_id = 'BP-1729'
+        _location_id_1 = '430406089232901'
+        _location_id_2 = '430406089232902'
+
         _iddata_url = 'ngwmn/iddata?request={}&agency_cd={}&siteNo={}'
-        self.well_log_url = urljoin(SERVICE_ROOT, _iddata_url.format('well_log', _agency_cd, _location_id))
-        self.wq_url = urljoin(SERVICE_ROOT, _iddata_url.format('water_quality', _agency_cd, _location_id))
-        self.sifta_url = COOP_SERVICE_PATTERN.format(site_no=_location_id)
-        self.site_loc_url = '/provider/{0}/site/{1}/'.format(_agency_cd, _location_id)
+        self.well_log_url = urljoin(SERVICE_ROOT, _iddata_url.format('well_log', _agency_cd, _location_id_1))
+        self.wq_url = urljoin(SERVICE_ROOT, _iddata_url.format('water_quality', _agency_cd, _location_id_1))
+        self.sifta_url = COOP_SERVICE_PATTERN.format(site_no=_location_id_1)
+        self.site_loc_url_1 = '/provider/{0}/site/{1}/'.format(_agency_cd, _location_id_1)
+
+        self.well_log_url_2 = urljoin(SERVICE_ROOT, _iddata_url.format('well_log', _agency_cd, _location_id_2))
+        self.wq_url_2 = urljoin(SERVICE_ROOT, _iddata_url.format('water_quality', _agency_cd, _location_id_2))
+        self.sifta_url_2 = COOP_SERVICE_PATTERN.format(site_no=_location_id_2)
+        self.site_loc_url_2 = '/provider/{0}/site/{1}/'.format(_agency_cd, _location_id_2)
 
         _stats_url = '/'.join(['ngwmn_cache', 'direct', 'json'])
-        self.site_info_url = '/'.join([SERVICE_ROOT, _stats_url, 'site-info', _agency_cd, _location_id])
-        self.stats_overall_url = '/'.join([SERVICE_ROOT, _stats_url, 'wl-overall', _agency_cd, _location_id])
-        self.stats_monthly_url = '/'.join([SERVICE_ROOT, _stats_url, 'wl-monthly', _agency_cd, _location_id])
+        self.site_info_url = '/'.join([SERVICE_ROOT, _stats_url, 'site-info', _agency_cd, _location_id_1])
+        self.stats_overall_url = '/'.join([SERVICE_ROOT, _stats_url, 'wl-overall', _agency_cd, _location_id_1])
+        self.stats_monthly_url = '/'.join([SERVICE_ROOT, _stats_url, 'wl-monthly', _agency_cd, _location_id_1])
+        self.site_info_url_2 = '/'.join([SERVICE_ROOT, _stats_url, 'site-info', _agency_cd, _location_id_2])
+        self.stats_overall_url_2 = '/'.join([SERVICE_ROOT, _stats_url, 'wl-overall', _agency_cd, _location_id_2])
+        self.stats_monthly_url_2 = '/'.join([SERVICE_ROOT, _stats_url, 'wl-monthly', _agency_cd, _location_id_2])
+
         self.mock_site_info_json = json.dumps(MOCK_SITE_INFO)
         self.mock_overall_json = json.dumps(MOCK_OVERALL_STATS)
         self.mock_monthly_json = json.dumps(MOCK_MONTHLY_STATS)
 
     @requests_mock.Mocker()
     def test_best_case(self, mocker):
+        id_of_expected_nested_location = b'430406089232901'
```

Comment:
This variables are confusingly named. How about calling it id1 and id2, set the values in the setup, and adjust the tests accordingly. Our use case for this issue happens to be "nested" sites. Naming them as such here just confuses the issue.

Reply:
Makes sense to me. Will do.